.circleci/config.yml

```diff
@@ -47,10 +47,12 @@ workflows:
     - lumigo-orb/print_orb_versions:
         lumigo_orb_version: *lumigo_orb_version
 
-    - test:
-        context:
+    - lumigo-orb/test:
+        context: 
           - common
           - nuget
+        name: lumigo-orb/test
```

Comment:
I think you want to keep this as it was, I dont know if the orb test job support .net